src/components/FeatureCards/FeatureCards.tsx

```diff
@@ -25,22 +31,38 @@ export const FeatureCards = () => {
           !disabledFeatureCards.includes(el.attributes.DisplayConditions?.id),
       );
     }
-    // trigger featureCardsFetched-filtering only once
+    //trigger featureCardsFetched-filtering only once
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [data, isSuccess]);
 
   useEffect(() => {
     if (Array.isArray(featureCardsFetched)) {
       !!featureCardsFetched.length &&
-        setFeatureCards(featureCardsFetched?.slice(0, 4));
+        setFeatureCards(featureCardsFetched?.slice(0, 2));
     }
   }, [featureCardsFetched]);
+
+  useEffect(() => {
+    if (Array.isArray(personalizedCardsFetched)) {
+      setPersonalizedFeatureCards(personalizedCardsFetched?.slice(0, 1));
+    }
+  }, [personalizedCardsFetched]);
+
```

Comment:
There is no need to use `useEffect` here and we can just use data received from hooks.

cc @dennyscode